faker/providers/person/ja_JP/__init__.py

```diff
@@ -1,4 +1,5 @@
 from .. import Provider as PersonProvider
+from collections import OrderedDict
```

Comment:
can you move this import to the top of the file, and separate std-library imports from relative imports with a new line?
```suggestion
from collections import OrderedDict

from .. import Provider as PersonProvider
```